makelib/golang.mk

```diff
@@ -132,22 +133,37 @@ endif
 GO_COMMON_FLAGS = $(GO_BUILDFLAGS) -tags '$(GO_TAGS)'
 GO_STATIC_FLAGS = $(GO_COMMON_FLAGS) $(GO_PKG_STATIC_FLAGS) -installsuffix static  -ldflags '$(GO_LDFLAGS)'
 
+GO_INIT_REQUIRED_TARGETS = go.vendor.lite
+GO_VENDOR_TARGET = go.vendor
+# switch for go modules
+ifeq ($(GO111MODULE),on)
+# no GOPATH is set, since go modules are being used
+export GOPATH= 
+export GO111MODULE=on
+
+GO_STATIC_FLAGS += -mod=vendor
+GO_INIT_REQUIRED_TARGETS = go.mod.verify
```

Comment:
does this just overwrite the value that was set in `GO_INIT_REQUIRED_TARGETS` earlier on line 136?

Reply:
yes, this happens when `GO111MODULE=on`

Reply:
ah yes, i missed that it was within an `ifeq`/`endif` block 😅